tools/packaging/kata-deploy/local-build/Makefile

```diff
@@ -9,6 +9,26 @@ MK_DIR := $(dir $(MK_PATH))
 # Verbose build
 V := 1
 
+ifeq ($(CROSS_BUILD),false)
```

Comment:
Hi @BbolroC ! The default behavior is the user not export `CROSS_BUILD=false` and this `ifeq` will fail, resulting on a wrong build for x86_64. You should set `CROSS_BUILD=false` if not exported.

Also I suggest to set `EXTRA_TARBALL` and `SERIAL_EXTRA_TARBALL` with the x86_64 targets by default. If `CROSS_BUILD=true` and `ARCH=s390x` then overwrite `EXTRA_TARBALL` and `SERIAL_EXTRA_TARBALL`.